cmake-modules/FolderList.cmake

```diff
@@ -12,6 +12,9 @@ macro(GetFolderList project)
     elseif(${project} STREQUAL KEYS)
         DownloadDepVersion(sdk/core azure-core 1.5.0)
         DownloadDepVersion(sdk/identity azure-identity 1.1.0)
+    elseif(${project} STREQUAL ADMINISTRATION)
+        DownloadDepVersion(sdk/core azure-core 1.5.0)
```

Comment:
Have you actually tested building keyvault-administration against azure-core 1.5.0? Should this instead be 1.8?

Reply:
yes